### DISCUSSION TITLE
This document requires 'TrustedHTML' assignment.

### DISCUSSION THREAD

**Discussion by mdfarooqs1111** [jquery/jquery#5442] [General]
![image](https://github.com/jquery/jquery/assets/88963108/bccb0afb-9640-4dde-b061-32a80ad0e34c)
![image](https://github.com/jquery/jquery/assets/88963108/1c393a9d-b91b-4d4b-b17e-db15ca3f3ab7)

**Comment by dmethvin** [comment]
https://stackoverflow.com/questions/61964265/getting-error-this-document-requires-trustedhtml-assignment-in-chrome

https://www.google.com/search?q=This+document+requires+%27TrustedHTML%27+assignment&oq=This+document+requires+%27TrustedHTML%27+assignment&gs_lcrp=EgZjaHJvbWUyCQgAEEUYORiABDIICAEQRRgnGDsyDQgCEAAYhgMYgAQYigUyDQgDEAAYhgMYgAQYigUyDQgEEAAYhgMYgAQYigUyBggFEEUYPTIGCAYQRRg9MgYIBxBFGDzSAQczNDhqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8

**Comment by mgol** [comment]
Trusted HTML support will arrive in jQuery 4.0.0.
